### ISSUE TITLE
Focused Launch: Focused styles should be different from selected styles

### ISSUE THREAD

**Issue by ciampo** [Automattic/wp-calypso#47741]
Let's focus first on https://github.com/Automattic/wp-calypso/issues/51175 and then re-evaluate this one.

#### Steps to reproduce

1. Create a new site and pick a free plan
2. Go to the block editor and click on the "Launch" button in the editor's header
3. Keep pressing the `tab` key to interact with the Focused Launch interface
4. Notice how focused and selected items are styled the same

#### What I expected

- [x] The user should be able to tab through the (non disabled) domain step items, and press the `space` or `enter` key to select those items — similarly to how interacting with the plan items in the Plan step works. (Done in https://github.com/Automattic/wp-calypso/pull/48172)
- [ ] Focused styles should be different from selected styles. If keyboard focus is moved onto the currently selected domain item, it should be clear to the user that the domain item is both selected and focused

**Comment by razvanpapadopol** [comment]
On hold until we do #51175

**Comment by youbrokesomething** [comment]
The focused launch is no longer active and has been replaced by other initiatives. I'm going to close this issue.
